index.html

```diff
@@ -0,0 +1,276 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Inti Raymi Music Festival</title>
+    <link rel='stylesheet' href='https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css'>
+    <link rel="stylesheet" href="normalize.css">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,300;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+
+  <section class="top-page">
+    <div class="session-container">
```

Comment:
Kindly use nav tag for your all menu item wrapper. You can use nav tag more than once. Check to link to learn more: https://stackoverflow.com/questions/4676388/multiple-nav-tags#:~:text=Yes%2C%20absolutely.,and%20footer%20tags%20sans%20penalty.

Reply:
Great, thanks for the suggestions! It's done